### PR TITLE
adds get_unsorted_spike_train method to waveclussortingextractor

### PR DESCRIPTION
**Pull request by ferchaure**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Hi Fernando.
This method have to implemented also for the base class with a return None (maybe) when it is not implemented.
```
def get_unsorted_spike_train(self):
    return None
```

**Comment by ferchaure** [comment]
Which is better, return None or raise a NotImplementedError on the baseextractor?

**Comment by alejoe91** [comment]
Maybe a NotImplementedError? Or an empty np.array.. @samuelgarcia what do you think?

**Comment by samuelgarcia** [comment]
Maybe the NotImplementedError is the wiser.

**Comment by samuelgarcia** [comment]
I think it is OK for me.
I will add the same for tridesclous soon.

File: spikeextractors/baseextractor.py [hunk 1]
```diff
@@ -274,6 +274,9 @@ def load_extractor_from_pickle(pkl_file):
             extractor._features = d['features']
         return extractor
 
+    def get_unsorted_spike_train(self):
```
Comment (alejoe91) [inline spikeextractors/baseextractor.py:277]: @ferchaure can you move this to the SortingExtractor class? This base extractor is a common class for recording and sorting extractors which handles allocating memory, dumping etc

Comment (ferchaure) [reply spikeextractors/baseextractor.py:277]: ok, sorry
should I include a _start_frame_ and _end_frame_ inputs?

Comment (alejoe91) [reply spikeextractors/baseextractor.py]: That would be perfect!

**Pull request merged**